metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -3,10 +3,11 @@
   (:require
    [clj-time.core :as t]
    [clojure.set :as set]
+   [cmr.common.api.context :refer (context->user-id)]
```

Comment:
why refer and not as?

Reply:
The function existed here before, so I thought just using refer. 
Since you don't like it this way. I will change it to use :as